<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>490</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="background">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>960</width>
        <height>540</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/bg.jpg</normaloff>:/bg.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>960</width>
        <height>540</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="MyWidget" name="memory" native="true">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>561</width>
        <height>331</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>20</y>
         <width>581</width>
         <height>321</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
      <widget class="QPushButton" name="btn_memory_min">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>0</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
     <widget class="MyWidget" name="disk" native="true">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>561</width>
        <height>331</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>20</y>
         <width>581</width>
         <height>321</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="diskDisplay"/>
      </widget>
      <widget class="QPushButton" name="btn_disk_min">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>0</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
     <widget class="fileWidget" name="file" native="true">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>561</width>
        <height>331</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <widget class="QPushButton" name="btn_file_min">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>0</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="desktop">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>computer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="background_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>960</width>
        <height>540</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/bg.jpg</normaloff>:/bg.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>960</width>
        <height>540</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>960</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QPushButton" name="winBar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>960</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/winBar.jpg</normaloff>:/winBar.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>961</width>
       <height>51</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_memory">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>3</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 170, 255);
        color: white;  
        border-radius: 10px;  border: 2px groove rgb(0, 170, 255);
        border-style: outset;</string>
     </property>
     <property name="text">
      <string>memoryShow</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_disk">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>3</y>
       <width>160</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
        color: white;  
        border-radius: 10px;  border: 2px groove gray;
        border-style: outset;</string>
     </property>
     <property name="text">
      <string>diskShow</string>
     </property>
    </widget>
    <widget class="QLabel" name="clock">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>0</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="128">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="128">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="127">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="128">
           <red>231</red>
           <green>231</green>
           <blue>231</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#e5e6dc;&quot;&gt;Time&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 188, 0);
        color: white;   
        border-radius: 10px;  border: 2px groove gray;
        border-style: outset;</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_file">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>3</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
        color: white;  
        border-radius: 10px;  border: 2px groove gray;
        border-style: outset;</string>
     </property>
     <property name="text">
      <string>cataloge</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyWidget</class>
   <extends>QWidget</extends>
   <header>mywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>fileWidget</class>
   <extends>QWidget</extends>
   <header>filewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
